numalogic/udfs/postprocess.py

```diff
@@ -63,6 +63,16 @@ def __init__(
         )
         self.postproc_factory = PostprocessFactory()
 
+    @staticmethod
+    def _calculate_unified_score(anomaly_scores, scoring_function):
+        if scoring_function == "max":
+            return np.max(anomaly_scores)
+        if scoring_function == "min":
+            return np.min(anomaly_scores)
+        if scoring_function == "mean":
+            return np.mean(anomaly_scores)
+        raise NotImplementedError(f"Unsupported loss function provided: {scoring_function}")
```

Comment:
Can you use something like `Literal`? https://docs.python.org/3.9/library/typing.html#typing.Literal

Reply:
This is interesting!

Reply:
Discussed.